docs/07-tools.md

```diff
@@ -173,9 +171,9 @@ Before Babel will actually compile your code, it needs to be configured. Create
 }
 ```
 
-There are a few unusual elements to this setup. First, we're setting `"modules": false`, otherwise Babel will convert our modules to CommonJS before Rollup gets a chance to do its thing, causing it to fail.
+There are a few unusual elements to this setup. First, we're setting "modules": false, otherwise Babel will convert our modules to CommonJS before Rollup gets a chance to do its thing, causing it to fail.
 
-Secondly, we're putting our `.babelrc` file in `src`, rather than the project root. This allows us to have a different `.babelrc` for things like tests, if we need that later – it's generally a good idea to have separate configuration for separate tasks.
+Secondly, we're putting our `.babelrc` file in `src`, rather than the project root. This allows us to have a different `.babelrc` for things like tests, if we need that later – See the [Babel documentation](https://babeljs.io/docs/en/config-files#project-wide-configuration) for more information on both project wide and file relative configuration. 
```

Comment:
Isn't that kind of a contradiction if we define a project-wide `babel.config` above but talk about "putting our `.babelrc`" somewhere here? Either we change this to be a `babel.config` here as well, but then the remaining text needs to be adjusted, or we stick with `.babelrc`, possibly with a `.json` extension?

My feeling also goes to the latter considering we are still placing it in `src`.

Reply:
great catch. addressed in 6c7015a